### ISSUE TITLE
Document our Working Mode

### ISSUE THREAD

**Issue by zcorpan** [w3c/aria-at#41]
I wrote a document that we could use as a starting point for discussion for how we work and handle disagreements in ARIA-AT CG. Hopefully, there won't be any disagreements, and we'll sail smoothly to our end goal, but if that doesn't happen I think it can be good to have an agreed upon framework.

This is a Google Doc for now, which is public and anyone can comment and make suggestions. For archive purposes, I have sent [a plain text copy to www-archive@w3.org](https://lists.w3.org/Archives/Public/www-archive/2020Feb/0003.html). If we want to have a document like this for the CG, I think we should move it to the wiki for this repository.

The document is here:
https://docs.google.com/document/d/1CXNRSDj1rFsVPtCu4YFww7I0OUEGRIXY6i_s453uaak/edit?usp=sharing

**Comment by zcorpan** [comment]
@mcking65 @mfairchild365 can you review this?

**Comment by mfairchild365** [comment]
Yep, I'll add comments in the doc. Might be good to discuss on the CG call too.

**Comment by zcorpan** [comment]
I've made changes based on feedback from @mcking65. Please review.

Also I've moved an idea about web content research to APG: https://github.com/w3c/aria-practices/issues/1348

A snapshot of the current revision of the document is also available [in HTML format in www-archive](https://lists.w3.org/Archives/Public/www-archive/2020Mar/att-0001/aria-at-cg-working-mode-2020-03-05.html).

**Comment by mcking65** [comment]
@zcorpan, this is coming along very well.

I'd like to make some more fairly significant revisions to the process section. I tried doing it in the google doc, and that just doesn't work for me. So, I am attaching a word doc with my proposal for the process section.

Additionally:
* I think it might read better if we change "AT Implementer" to "AT Developer".
* I think we need to separate APG Task Force from ARIA WG in the roles even though the TF is part of the WG, there are separate repos and thus separate places to file issues.

I'd like to get a final draft of the roles and process before going on to make revisions to the subsequent sections.
[Process section of ARIA-AT CG Working Mode.docx](https://github.com/w3c/aria-at/files/4302662/Process.section.of.ARIA-AT.CG.Working.Mode.docx)

**Comment by zcorpan** [comment]
Thank you, @mcking65. I have incorporated your edits in the google doc. I've also changed to AT developer and added the APG Task Force role.

**Comment by zcorpan** [comment]
@mcking65 

> (Test developer) Implement the test in WPT format

At this stage, these tests aren't in web-platform-tests, and don't have the same format. So I think we shouldn't use "WPT format" wording, to avoid confusion.

**Comment by zcorpan** [comment]
I have added a Definitions section based on https://github.com/w3c/aria-at/issues/102#issuecomment-596173899 - and added a few more definitions. I've gone through the working mode document to make sure it's using these terms as appropriate.

I also added new steps in the process for handling feedback.

**Comment by mcking65** [comment]
@zcorpan commented:

> > (Test developer) Implement the test in WPT format
> 
> At this stage, these tests aren't in web-platform-tests, and don't have the same format. So I think we shouldn't use "WPT format" wording, to avoid confusion.

They need to be in wpt format and on the aria-at web site site in the review section before AT devs can review them. And, per our earlier discussion, this is where it is important there is some level of GitHub integration so the AT dev can very quickly and easily create a useful issue with all the right info.

Perhaps you are thinking of internal peer reviews among test devs. You might add that as a subbullet under #5 as part of test development. Currently, we are doing the peer reviews with in the CG in CSV format only because we do not yet have a test composer. That is an implementation detail that the document does not have to address.

**Comment by zcorpan** [comment]
I have now moved this to the wiki: https://github.com/w3c/aria-at/wiki/Working-Mode

**Comment by zcorpan** [comment]
As discussed in the call today, I've updated some terms:

previous term | current term
---|---
test run | test plan run
test run suite | test plan run suite
testing round | full test cycle

I also added "partial test cycle", which is an arbitrary subset of full test cycle. This could be used for re-running a specific test plan or a specific browser/AT combination.
